packages/mui-material/src/Tabs/Tabs.test.js

```diff
@@ -360,6 +364,19 @@ describe('<Tabs />', () => {
             ].join('\n'),
           ]);
         });
+
+        it('should not warn if the whole Tabs is hidden', function test() {
+          if (isJSDOM) {
+            this.skip();
+          }
+          expect(() => {
+            render(
+              <Tabs value="demo" sx={{ display: 'none' }}>
```

Comment:
```suggestion
              <Tabs value="demo" style={{ display: 'none' }}>
```

For consistency with other tests.